### ISSUE TITLE
Package archived on CRAN

### ISSUE THREAD

**Issue by astamm** [stanfill/rotationsC#8]
Hi Bryan,
I see that once more the package has been archived by CRAN folks due to non fixing some issues in time.
I really think the features of this package are quite nice and should be maintained to ensure accessibility on CRAN.
I can again help here but chances are it will happen again later as well.
Do you have time to maintain the package?
If not, would you be open to discussion as to switch maintainer?

**Comment by stanfill** [comment]
Submitted to CRAN yesterday.  Passed auto inspection waiting for manual.  Thanks for the remainder.
